src/plugins/HistoryStatePlugin.js

```diff
@@ -35,6 +36,10 @@ export class HistoryStatePlugin extends BaPlugin {
 	}
 
 	_updateHistory() {
-		this._environmentService.getWindow().history.replaceState(null, '', this._shareService.encodeState());
+		const encodedState = this._shareService.encodeState();
+		if (this._currentEncodedState !== encodedState) {
+			this._environmentService.getWindow().history.replaceState(null, '', this._shareService.encodeState());
```

Comment:
iis a additional call of `this._shareService.encodeState()` needed?  
Otherwise `this._environmentService.getWindow().history.replaceState(null, '', [encodedState]);` would be more readable for me.

Reply:
Thx